pyproject.toml

```
@@ -24,6 +24,7 @@ requires-python = ">=3.9"
 dependencies = [
     "caput @ git+https://github.com/radiocosmology/caput.git",
     "caput[compression] @ git+https://github.com/radiocosmology/caput.git",
+    "caput[fftw] @ git+https://github.com/radiocosmology/caput.git",
```

Does pyproject allow you to combine these?
```suggestion
    "caput[compression,fftw] @ git+https://github.com/radiocosmology/caput.git",
```

---

Let's find out